index.html

```diff
@@ -9,13 +9,16 @@
         <title>Dinner Decider</title>
         <meta name="description" content="">
         <meta name="viewport" content="width=device-width, initial-scale=1">
-        <link rel="stylesheet" href="">
```

Comment:
Duplicate stylesheet referenced / not used

Reply:
I will remove that.